### PR TITLE
Clean up indexer to remove support for embeddings generation & add initial path for Vespa support

### PR DESCRIPTION
**Pull request by joel-wright**
Ticket for reference:

https://linear.app/climate-policy-radar/issue/PDCT-335/remove-embeddings-code-from-the-indexer

### CODE REVIEW

File: cli/run.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -e
 
-python -m cli.text2embeddings --s3 --device=cpu "${EMBEDDINGS_INPUT_PREFIX}" "${INDEXER_INPUT_PREFIX}"
\ No newline at end of file
+python -m cli.index_data --s3 "${INDEXER_INPUT_PREFIX}"
```
Comment (THOR300) [inline cli/run.sh:4]: Do we even need a `run.sh` here anymore?

**Review by THOR300 — APPROVED** [review]
LGTM - I guess the only real way for us to safely test is the spin up an opensearch instance in sandbox or staging and run an ingest.

**Review by THOR300 — APPROVED** [review]

**Review by THOR300 — APPROVED** [review]
Nice one!

File: cli/run.sh (hunk 1 above)
Comment (joel-wright) [reply cli/run.sh:4]: probably not, but that can be done in a follow-up I reckon

**Review by THOR300 — APPROVED** [review]
DAL tag bump LGTM

**Pull request merged**